Sources/Basics/Environment/Environment.swift

```diff
@@ -19,6 +19,8 @@ import Musl
 #elseif os(Windows)
 import CRT
 import WinSDK
+#elseif canImport(Android)
+import Android
```

Comment:
Can we use `Bionic` here instead? Note that Darwin is using `Darwin.C` and Windows is using `CRT`. Of course if we need more than the C interfaces, `Android` is fine, but would bee nice to keep this scoped to the same level as the other platforms.

Reply:
I'll check if that's enough.

Reply:
Done